### PR TITLE
Gromacs 5

### PR DESCRIPTION
**Pull request by wpoely86**
Needs hpcugent/easybuild-easyblocks#513

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux debian 7.7, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/cf1f91c8e193f493828d for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux debian 7.7, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/181e007947a923c79e76 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux debian 7.7, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/37dd1821a07cb89aa131 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux debian 7.7, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/e554beff795e623c6a13 for a full test report.

**Comment by boegel** [comment]
@wpoely86: can you look into a test report on top of current https://github.com/hpcugent/easybuild-easyblocks/pull/513 for this? I want to get this included in EB 2.1.1

File: easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.2-ictce-7.1.2-mt.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'ictce', 'version': '7.1.2'}
+toolchainopts = {'openmp': True, 'usempi': False}
+
+source_urls = [
+    'ftp://ftp.gromacs.org/pub/gromacs/',
+    'http://gerrit.gromacs.org/download/',
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.2-ictce-7.1.2-mt.eb]: this URL doesn't exist (anymore)

File: easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.2-ictce-7.1.2-mt.eb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+]
+
+sources = [
+    SOURCELOWER_TAR_GZ,
+# seems to have disappeared?
+#    'regressiontests-5.0.2.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.2-ictce-7.1.2-mt.eb]: remove this, collapse this block to:

```python
source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
sources = [SOURCELOWER_TAR_GZ]
```

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/462c7961fb7653aeefd9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 20 out of 21 (21 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ab744da39329aefd8366 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/4691efa29eedf0482767 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3ca27981f4a4ae7b83c6 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/b924e1847c4f9be1af3d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 20 out of 20 (21 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4db020f79ec7ef3f1651 for a full test report.

**Comment by boegel** [comment]
Well-tested on top of https://github.com/hpcugent/easybuild-easyblocks/pull/513, good to go.

Thanks @wpoely86!

**Pull request merged**